doc/active_docs/Service Management API (on-premises).json

```diff
@@ -1,693 +0,0 @@
-{
```

Comment:
I don't think you c an remove this one

Reply:
Is this like this file will be used on-premise

But there we two files and one with no environment mentioned so considering that a generic one I preferred to  keep that and remove this

Reply:
I'd say it's being used from here:

https://github.com/3scale/porta/blob/df1b415ad931e42ef5d9f51b88223076a74b6319/app/controllers/api_docs/services_controller.rb#L57

Also, the other files you deleted belong to the `:new_provider_documentation` rolling update. If you want to do a proper clean you need to remove the code for all the rolling update, check how the controller looks for the rolling update before deciding which view it loads:

https://github.com/3scale/porta/blob/df1b415ad931e42ef5d9f51b88223076a74b6319/app/controllers/provider/admin/api_docs_controller.rb#L7

I think the cleanup should be done in another PR, better not mix things.

Reply:
this sounds a better idea to clean it up later

Reply:
It's OK to clean up in a different PR, but I think this PR should not touch the `*-s20.json` files, and should just leave them as they are, because:
1. They are not used unless `new_provider_documentation: true`
2. If `new_provider_documentation: true` the feature becomes broken
3. This is confusing :)

So, I think we need to restore the original contents of these files, and we can remove them later in a different PR.